docarray/array/storage/elastic/backend.py

```diff
@@ -40,7 +40,7 @@ class ElasticConfig:
     es_config: Dict[str, Any] = field(default_factory=dict)
     index_text: bool = False
     tag_indices: List[str] = field(default_factory=list)
-    batch_size: int = 64
+    bulk_config: Dict[str, Any] = field(default_factory=dict)
```

Comment:
is there a way this can be made non breaking change? can we detect if batch size is added?

Reply:
sure, will add it

Reply:
done, if the bulk config is provided, the configuration from bulk config will take priority